Index: u-boot-2026.01/dts/upstream/src/arm64/airoha/en7581.dtsi
===================================================================
--- u-boot-2026.01.orig/dts/upstream/src/arm64/airoha/en7581.dtsi
+++ u-boot-2026.01/dts/upstream/src/arm64/airoha/en7581.dtsi
@@ -150,7 +150,7 @@
 			interrupts = <GIC_PPI 9 IRQ_TYPE_LEVEL_LOW>;
 		};
 
-		spi@1fa10000 {
+		snfi: spi@1fa10000 {
 			compatible = "airoha,en7581-snand";
 			reg = <0x0 0x1fa10000 0x0 0x140>,
 			      <0x0 0x1fa11000 0x0 0x160>;
